### PR TITLE
added subscription support

### PR DESCRIPTION
**Pull request by matthewtanner91**

### CODE REVIEW

File: setup.py
Comment (praves77) [inline setup.py:1]: @matthewtanner91 it needs version bump up to publish correctly.. probably line number is 42??

File: tests/settings.py [hunk 1]
```diff
@@ -90,7 +90,7 @@
 ]
 
 MOESIF_MIDDLEWARE = {
-    'APPLICATION_ID': 'Your Moesif Application Id',
+    'APPLICATION_ID': 'eyJhcHAiOiI1NjU6NDYzIiwidmVyIjoiMi4xIiwib3JnIjoiNjI4OjM2MCIsImlhdCI6MTcwNjc0NTYwMH0.KTkq3jzf7vTTDeszWF39n8EiDNM5-DX0ld1ASvinoi4',
```
Comment (praves77) [inline tests/settings.py:93]: same cleanup needed

File: setup.py
Comment (matthewtanner91) [reply setup.py:1]: from version='2.3.8', to version='2.3.9' ?

Comment (matthewtanner91) [reply setup.py:1]: changed

File: tests/settings.py (hunk 1 above)
Comment (matthewtanner91) [reply tests/settings.py]: fixed

**Review by praves77 — APPROVED** [review]

**Pull request merged**